src/main/java/calculator/controller/InputCalculator.java

```diff
@@ -0,0 +1,16 @@
+package calculator.controller;
+
+import calculator.model.Input;
+import java.util.List;
+
+public class InputCalculator {
+
+    public static int calculateResult(Input input) {
+        List<Integer> numbers = input.getNumbers();
+        int sum = 0;
+        for (Integer number : numbers) {
```

Comment:
for문 대신 스트림 API를 사용해볼 수 있을 것 같습니다. `mapToInt`, `IntStream` 키워드로 내용을 확인해보셔도 좋을 것 같아요.

Reply:
사실 아직 stream을 사용하는데에 미숙해서 for문을 사용하고있습니다.... 앞으로 꾸준히 학습해나가야할 것 같아요

Reply:
그런데 한가지 궁금한점이 있습니다.
stream의 좋은점이 어떤것이라고 생각하시나요?
stream을 사용하면 for문을 효과적이게 축약할 수는 있으나 역할의 분리가 힘들어지는 부작용의 위험도 있지 않을까요?
의견이 궁금합니다!

Reply:
제가 생각한 `calculateResult`의 역할은 `Input`에 있는 요소들을 전부 더한 값을 반환한다라고 생각했습니다.
```suggestion
        return input.getNumbers()
                .stream()
                .mapToInt(Integer::intValue)
                .sum();
```
이처럼 스트림을 사용하면 컬렉션 내 데이터들에 신경을 쓰지 않아도 된다는 장점이 있다고 생각합니다.
`List<Integer>`를 참조하는 변수(`numbers`)나 총합을 저장하는 `sum`이라는 변수를 생성하지 않아도 되며, 마지막 `.sum()`을 통해 컬렉션 내 데이터들의 총합을 반환한다는 의미를 전달할 수 있습니다.

무조건 스트림이 정답도 아니고 상황에 따라 For문이 필요한 경우도 있습니다. 예를 들어, 컬렉션 내 데이터들을 특정 조건에 따라 변환하거나 별도의 로직을 처리해야 한다면 For문을 사용하는게 더 가독성이 높을 수도 있겠습니다.